src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -140,6 +141,20 @@ public function testDenormalizeWithNestedAttributesWithoutMetadata()
         $this->assertNull($test->notfoo);
     }
 
+    public function testDenormalizeWithSnakeCaseNestedAttributes()
+    {
+        $factory = new ClassMetadataFactory(new AnnotationLoader(new AnnotationReader()));
+        $normalizer = new AbstractObjectNormalizerDummy($factory, new CamelCaseToSnakeCaseNameConverter());
+        $serializer = new Serializer([$normalizer]);
+        $data = [
+            'one' => [
+                'two_three' => 'fooBar',
+            ],
+        ];
+        $test = $serializer->denormalize($data, SnakeCaseNestedDummy::class, 'any');
```

Comment:
```suggestion
        $test = $normalizer->denormalize($data, SnakeCaseNestedDummy::class, 'any');
```
So that you can get rid of the serializer instantiation.